pr_agent/tools/pr_similar_issue.py

```diff
@@ -194,8 +190,8 @@ def _update_index_with_issues(self, issues_list, repo_name_for_index, upsert=Fal
 
             issue_str, comments, number = self._process_issue(issue)
             issue_key = f"issue_{number}"
-            username = issue.user.login
-            created_at = str(issue.created_at)
+            username = self.git_provider.get_username(issue, self.issue_url.split('=')[-1])
+            created_at = self.git_provider.get_issue_created_at(issue)
             if len(issue_str) < 8000 or \
```

Comment:
**Suggestion:** The `except` clause is empty and doesn't handle any specific exceptions. Consider adding proper exception handling or removing the try-except block if it's not necessary. [error handling, importance: 8]